homeassistant/components/modbus.py

```diff
@@ -26,6 +26,7 @@
 CONF_STOPBITS = 'stopbits'
 CONF_TYPE = 'type'
 CONF_PARITY = 'parity'
+CONF_TIMEOUT = 'timeout'
```

Comment:
This one as well.

Reply:
So could be better to move to `const.py` also:

```python
CONF_BAUDRATE = 'baudrate'
CONF_BYTESIZE = 'bytesize'
CONF_STOPBITS = 'stopbits'
CONF_PARITY = 'parity'
```

because they are settings related to serial port and usable by all components that need serial port too.

Reply:
At the moment `modbus` seems to be the only one that uses those. So, it doesn't make sense to move. We will move them when more platforms/components start to require those constants.